src/edge/static/edge/src/partials/genome-detail.html

```diff
@@ -8,9 +8,12 @@
   </div>
 </header>
 
+
 <div class="container">
 
 <h3>
+  <div><a class="btn btn-primary" href="/edge/genomes/{{ genome.id }}/export">Export to GFF</a>
```

Comment:
These two lines should be:
<div>
  <a class="btn btn-primary" href="/edge/genomes/{{ genome.id }}/export">Export to GFF</a>
</div>
just to keep the style consistent.